parsing/dialects/postgresqlBase/dialects/postgresql/config.fmpp

```diff
@@ -458,6 +458,10 @@ data: {
       "InfixCast"
     ]
 
+    # Partition expressions
```

Comment:
nit: I think this comment could be removed in all the fmpp files since the name describes it

Reply:
Since you mentioned it, I figure we could go the other way to make it more descriptive. So I made them more descriptive in all fmpp files in https://github.com/googleinterns/calcite/pull/212/commits/0c3bc69d9374d1032384984d6aaee2356f6d142e. Please let me known what you think about this change. Would you say removing it would be a better choice?

Reply:
I think either way is fine actually considering these will be removed once the the #211 is merged

Reply:
You are right, the fmpp files are not going to be important anymore. Thanks for reminding me of that change.